app/views/shared/_navbar.html.erb

```diff
@@ -20,3 +22,73 @@
     </nav>
   </div>
 </div>
+
+<!-- I know, bad style. Should be moved to js-file later -->
+
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+<script>
```

Comment:
You're even using the CDN instead of installing the library in our package.json 😅 
In this project you should always add dependencies by using yarn, not by adding CDNs

Reply:
As I've said in previous posts: 

1. For the process of coding, I'm writing everything in one file, to write it later in the files, where it belongs to. This goes faster for me. 
2. About the jscript: I imported it, when I wanted to try it. It does not work for other reasons and I don't know why. You wanted to have the current file, I gave it to you and left the script in the html.erb, where it works. :) 

...so, yes, Will write it from scratch and solve this problem later.

Reply:
There's a difference between writing javascript inside the html file and using a cdn instead of a Node Package.
You should never use a cdn in this project. If you can't make it work by installing the node package then you don't do it. That's what I meant.
As far as your preference to write javascript inside the html first before moving it to a dedicated file I would recommend to stop doing that asap. It does not matter to me as long as you remove everything before making your PR but it's a bad habit and the longer you wait the more difficult it will become to get rid of